source/Sashimi.Tests.Shared/Server/ActionHandlerTestBuilder.cs

```diff
@@ -92,7 +92,7 @@ public ActionHandlerTestBuilder<TCalamariProgram> WithAssert(Action<TestActionHa
             return this;
         }
 
-        public TestActionHandlerResult Execute(bool assertWasSuccess = true, bool runInProc = true)
+        public TestActionHandlerResult Execute(bool assertWasSuccess = true, bool runOutOfProc = false)
```

Comment:
Actually this should be saving and restoring the previous state of "Test_Calamari_InProc_OutProc_Override", otherwise this will make the next test run `InProc` if the environment variable value was `OutProc` to begin with.

Also, if you make `TestCalamariCommandBuilder.InProcOutProcOverride` public then you can reference the constants there.

Reply:
Fixed